### ISSUE TITLE
support device profiles

### ISSUE THREAD

**Issue by GlenDC** [plabayo/rama#20]
- inmem db using <https://github.com/plabayo/venndb>
- create profiles (bin) file using #12 
- support in web client (#16)
- layer support

Desktop only for now.

**Comment by GlenDC** [comment]
Blocked by https://github.com/plabayo/rama/issues/64

**Comment by GlenDC** [comment]
UserAgentProfile:

- browser
- browser_version
- os
- platform
- http_ua
- http_ja4h
- http_fetch_mode
- http_resource_type
- http_version
- tls_ja4

UserAgentProfileId = hash(UserAgentProfile)

HttpProfile:

- ja4h (== id)
- http_headers
- http_pseudo_headers

TlsProfile:

- ja4 (== id)
- client_hello (filtered to what we can and need to emulate)

**Comment by GlenDC** [comment]
For storage we can simply use managed redis via fly.

**Comment by GlenDC** [comment]
> inmem db using https://github.com/plabayo/venndb

custom in-mem db instead

> create profiles (bin) file using

will be done via a python script instead

> https://github.com/plabayo/rama/issues/12
> support in web client (https://github.com/plabayo/rama/issues/16)

closed since

> layer support

added as emulate layer, in tandem with `UserAgent` context variable

> Desktop only for now.

no longer relevant

**Comment by GlenDC** [comment]
> For storage we can simply use managed redis via fly.

will be postgres instead, no reason for redis here
